apps/verticalhome/js/clock.js

```diff
@@ -120,17 +127,16 @@ function Clock() {
    * @memberOf Clock
    */
   this.start = function cl_start() {
-    var date = new Date();
-    var self = this;
 
-    this.refreshClock(date);
+    var date = new Date();
+    window.dispatchEvent(new CustomEvent('tick'));
 
     if (this.timeoutID == null) {
       this.timeoutID = window.setTimeout(function cl_setClockInterval() {
 
         if (self.timerID == null) {
```

Comment:
As I can see you have deleted line with ``self`` declaration (line 124 it was), so it seems to me, that this condition will be ``true`` always (``(undefined == null) - true``, BTW with '``===``' you could detect problems before PR ;) ) and following line could cause error.